ci/requirements-py35-min.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
     - pytest-cov
     - pytest-mock
     - pytest-timeout
+    - pytest-rerunfailures
```

Comment:
does this also need to move to the pip section?

Reply:
Do you remember where this is referenced? I didn't notice it being called by any of the conda versions.

Reply:
Sorry, I don't understand your comment. My understanding is that `pytest-rerunfailures` is not available for python 3.5 in conda, so I'm surprised to see it here in the conda section of the `py35-min` configuration.

Reply:
You're right, it should be moved to the pip section but could the entire file be deleted?

My question is basically what the use is of the file this requirements-py35-min.yml file? I've only run across the use of the .yml files in the "ci" directory in the "azure-pipelines.yml" file where they are used to create the environment for the test via the line:
`  - script: conda env create --quiet --file ci/requirements-py$(python.version).yml`

There's no listed py35-min version so this file probably isn't used. This is corroborated by there being no errors  caused by it. 

Not necessarily pertinent to this PR but tangentially relevant.

Reply:
I'm not sure if it's redundant at this point, but it is used on Travis https://travis-ci.org/pvlib/pvlib-python/jobs/646624536?utm_medium=notification&utm_source=github_status

let's move the line to the pip section for this PR and consider whether the file should be removed in a separate issue

Reply:
Thanks for pointing that out. I haven't really touched the Travis system so it's good to know this wrinkle. 

I'll move that over and confirm Travis is happy.